lib/cookie.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+class Cookie
+  attr_reader :setcookie_string
+  def initialize(str)
+    @setcookie_string = str
+  end
+
+  def to_s
+    "ORA_WX_SESSION='#{ora_wx_session_id}';portal=#{portal_id}"
+  end
+
+  def ora_wx_session_id
```

Comment:
Is this really meant to be part of the public interface? If so I suspect you need to document what it actually is.